docs/pages/database-access/guides/mongodb-self-hosted.mdx

```diff
@@ -133,24 +133,35 @@ Create the secrets:
   authority and `mongo.crt` with the generated certificate and key pair. You will
   need these files to enable mutual TLS on your MongoDB server.
   </TabItem>
-  <TabItem label="Replica set">
-  When connecting to a MongoDB replica set, sign certificates for each member
-  using the hostnames they're accessible at.
-
-  For example, if the first member is accessible at `mongo1.example.com` and
-  the second at `mongo2.example.com`, run:
-
+  <TabItem label="Replica Set">
+  You will need to provide a certificate/key pair for each MongoDB server using 
+  the hostnames they are accessible at, and then add the Teleport Database 
+  Client CA as an additional trusted CA.
+  
+  Creating certificate/key pairs using your own CA is out of this guide's scope.
```

Comment:
As with my first comment, I'd recommend making this part of the prerequisites where we require either a standalone or replica set cluster. Otherwise, a user planning to follow this guide quickly by copying and pasting commands will run into friction.

Reply:
done [665d6a1](https://github.com/gravitational/teleport/pull/36260/commits/665d6a1141ea872336d9434c88734c31f2b300da)